### PR TITLE
CLDR-14416 fix headers, dates, and status blurb for revision 63, cldr v39

### PR DESCRIPTION
**Pull request by rickmcgowan**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14416
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Review by srl295 — APPROVED** [review]

File: docs/ldml/tr35.md [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
-<tr><td>Date</td><td>2021-02-22</td></tr>
-<tr><td>This Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-62/tr35.md">https://www.unicode.org/reports/tr35/tr35-61/tr35.md</a></td></tr>
-<tr><td>Previous Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-61/tr35.md">https://www.unicode.org/reports/tr35/tr35-60/tr35.md</a></td></tr>
+<tr><td>Date</td><td>2021-04-06</td></tr>
+<tr><td>This Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-63/tr35.md">https://www.unicode.org/reports/tr35/tr35-63/tr35.md</a></td></tr>
+<tr><td>Previous Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-61/tr35.md">https://www.unicode.org/reports/tr35/tr35-61/tr35.md</a></td></tr>
 <tr><td>Latest Version</td><td><a href="https://www.unicode.org/reports/tr35/">https://www.unicode.org/reports/tr35/</a></td></tr>
 <tr><td>Corrigenda</td><td><a href="http://unicode.org/cldr/corrigenda.html">http://unicode.org/cldr/corrigenda.html</a></td></tr>
 <tr><td>Latest Proposed Update</td><td><a href="https://www.unicode.org/reports/tr35/proposed.md">https://www.unicode.org/reports/tr35/proposed.md</a></td></tr>
```
Comment (srl295) [inline docs/ldml/tr35.md:14]:
```suggestion
<tr><td>Latest Proposed Update</td><td><a href="https://unicode-org.github.io/cldr/ldml/tr35.html">https://unicode-org.github.io/cldr/ldml/tr35.html</a></td></tr>
```

File: docs/ldml/tr35.md [hunk 2]
```diff
@@ -6,15 +6,15 @@
 <table><tbody>
 <tr><td>Version</td><td>39</td></tr>
 <tr><td>Editors</td><td>Mark Davis (<a href="mailto:markdavis@google.com">markdavis@google.com</a>) and <a href="tr35.md#Acknowledgments">other CLDR committee members</a></td></tr>
-<tr><td>Date</td><td>2021-02-22</td></tr>
-<tr><td>This Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-62/tr35.md">https://www.unicode.org/reports/tr35/tr35-61/tr35.md</a></td></tr>
-<tr><td>Previous Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-61/tr35.md">https://www.unicode.org/reports/tr35/tr35-60/tr35.md</a></td></tr>
+<tr><td>Date</td><td>2021-04-06</td></tr>
+<tr><td>This Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-63/tr35.md">https://www.unicode.org/reports/tr35/tr35-63/tr35.md</a></td></tr>
+<tr><td>Previous Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-61/tr35.md">https://www.unicode.org/reports/tr35/tr35-61/tr35.md</a></td></tr>
```
Comment (srl295) [inline docs/ldml/tr35.md:11]:
```suggestion
<tr><td>This Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-63/tr35.html">https://www.unicode.org/reports/tr35/tr35-63/tr35.html</a></td></tr>
<tr><td>Previous Version</td><td><a href="https://www.unicode.org/reports/tr35/tr35-61/tr35.html">https://www.unicode.org/reports/tr35/tr35-61/tr35.html</a></td></tr>
```

**Review by srl295 — DISMISSED** [review]

**Review by srl295 — APPROVED** [review]

**Review by srl295 — APPROVED** [review]

**Comment by pedberg-icu** [comment]
This PR was incorrectly targeted to master instead of to the maint/maint-39 branch. Now what to do? Other recent chnages to the spec were all targeted to the maint/maint-39 branch and may conflict with these changes. Can a new PR with these changes be created against the maint/maint-39 branch?? MY BRS tasks for tonight assumed the spec work on maint/maint-39 would be done.

**Comment by pedberg-icu** [comment]
> ... Now what to do? ... Can a new PR with these changes be created against the maint/maint-39 branch?? ...

Wel, let me see if I can cherry-pick these to the maint branch. That is NOT what we were supposed to be doing. Hope that does not mess up the merge-back from maint to master.

**Comment by pedberg-icu** [comment]
Oh I see, this has not been merged yet. Well I will just replace this with a PR against the maint/maint-39 branch that does the same thing.

**Comment by pedberg-icu** [comment]
DO NOT MERGE this.

**Comment by pedberg-icu** [comment]
This is replaced by https://github.com/unicode-org/cldr/pull/1191 which makes basically the same changes but is based on and targets the maint/maint-39 branch.

**Pull request closed without merging**